docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,22 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+Learn how to integrate with Eclipse:
+
+* [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+* [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+* [[Import an existing sbt build into Eclipse|EclipseSbt]]
+    
+Learn how to integrate with Intellij:
+
+* [[Import an existing Maven project into Intellij|intellijMaven]]
+* [[Import an existing sbt build into Intellij|IntellijSbtJava]]
+    
+Learn more about Lagom:
+
+* [[Core concepts|CoreConcepts]]
+* [[Using the service locator|ServiceLocator]]
+* [[Using the message broker|MessageBrokerAPI]]
+* [[Using publish-subscribe|PubSub]]
+* [[Using persistent entities|PersistentEntity]]
```

Comment:
I didn't think the reader would be confused, because the next link does go to the "Core Concepts" section. And, I felt that for a logical flow, design philosophy should come first (I moved it from the welcome section because it goes into important details that are really core concepts). However, I've changed it back to list the Core Concepts file first in the section. This will be only temporary as this whole section will be changing.